ansible/roles/master/tasks/ha_master.yml

```diff
@@ -0,0 +1,82 @@
+- name: Generic | Install kubernetes node
+  action: "{{ ansible_pkg_mgr }}"
+  args:
+    name: kubernetes-node
+    state: latest
+  notify:
+    - restart daemons
+  when: not is_coreos
+
+- name: CoreOS | Get Systemd Unit Files for kubelet and kube-proxy
```

Comment:
Does ha req proxy on the master? I thought it was kubelet. If so, pls remove proxy on master.